modules/addStaticOnLocalStorage.js

```diff
@@ -0,0 +1,117 @@
+
+
+                for (let i = 1; i < q; i++) {
+                    if (document.querySelector(`#chrono_list_content > div.b-chrono-items-container.js-chrono-items-container > div > div:nth-child(${i}) > div.r-col > h2 > a > span`) !== null) {
+                        let checker = document.querySelector(`#chrono_list_content > div.b-chrono-items-container.js-chrono-items-container > div > div:nth-child(${i}) > div.r-col > h2 > a `)
+                            .textContent //полуучает текст
+                            .trim() //Удаляет пробелы "с краёв"
+                            .split(` `) //Правращает в масив по разделителю " "
+                        let stringValueCounter = 0
+                        if (stringValueCounter < 2) {
+                            for (let i = 0; i < checker.length; i++) {
+                                if (checker[i] === `О` ||
+                                    checker[i] === `принятии` ||
+                                    checker[i] === `производству`) { stringValueCounter++ }
+                            }
+                        }
+
+                        if (stringValueCounter > 2) {
+                            sostav = document.querySelector("#chrono_list_content > div.b-chrono-items-container.js-chrono-items-container > div > div:nth-child(" + i + ") > div.r-col > h2 > span > p:nth-child(1)")
+                                .textContent
+                                .trim() //Удаляет пробелы "с краёв"
+                                .split(` `) //Правращает в масив по разделителю " "
+                                .pop() //"вырезает" последний элемент масива
+                        } else if (sostav !== ' состав неопределен') {
+                            // console.log(`Состав:`, sostav, `идем дальше`)
+                        }
+                        else { sostav = ' состав неопределен' }
+                    }
+                }
+
+                sostavAtPopup(sostav)
+
+                let params = {
+                    saveScanStat: saveScanStat,
+                    sostav: sostav,
+                    docId: docId,
+                    a40: a40,
+                }
+                resolve(params)
+
+            } else if (mutationRecords.length === 1) {
+                return console.log(`Атрибуты изменились`, mutationRecords, `- Закрытие модального окна`)
+            }
+        });
+        observer.observe(elem, {
+            attributes: true,
+            childList: true
+        })
+    })
+    promise.catch((err) => { console.error(`Error: `, err) })
+        .then((params) => {
+
+            //Срабатывает прии НАЖАТИИ на "Сохранить"
+            params.saveScanStat.onclick = () => {
+                let now = new Date()
+                let dataForLS = {}
+                dataForLS = JSON.parse(localStorage.getItem(`sostav${params.sostav}`))
+                if (dataForLS == null) {
```

Comment:
you can reverse this logic of checking :)
check for existence ;)
`const dataForLS = JSON.parse(localStorage.getItem(`sostav${params.sostav}`));`
and then `if (dataForLS !==null)`